src/OpenTelemetry.AutoInstrumentation.Native/netfx_assembly_redirection.h

```diff
@@ -49,15 +49,15 @@ void CorProfiler::InitNetFxAssemblyRedirectsMap()
         { L"OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule", {1, 9, 0, 46} },
         { L"OpenTelemetry.Instrumentation.GrpcNetClient", {1, 9, 0, 44} },
         { L"OpenTelemetry.Instrumentation.Http", {1, 9, 0, 41} },
-        { L"OpenTelemetry.Instrumentation.Process", {0, 5, 0, 49} },
+        { L"OpenTelemetry.Instrumentation.Process", {0, 5, 0, 176} },
         { L"OpenTelemetry.Instrumentation.Quartz", {1, 0, 0, 50} },
         { L"OpenTelemetry.Instrumentation.Runtime", {1, 9, 0, 57} },
         { L"OpenTelemetry.Instrumentation.SqlClient", {1, 9, 0, 43} },
         { L"OpenTelemetry.Instrumentation.Wcf", {1, 0, 0, 47} },
-        { L"OpenTelemetry.Resources.Azure", {1, 0, 0, 52} },
+        { L"OpenTelemetry.Resources.Azure", {1, 0, 0, 172} },
         { L"OpenTelemetry.Resources.Host", {0, 1, 0, 139} },
```

Comment:
Can you double check if this has not changed?

Reply:
It is not changed in this PR. What is more, our pipeline will fail if the file is wrongly generated.